pkg/gui/keybindings.go

```diff
@@ -745,27 +587,6 @@ func (self *Gui) GetInitialKeybindings() ([]*types.Binding, []*gocui.ViewMouseBi
 			Description: self.c.Tr.LcScrollRight,
 			Tag:         "navigation",
 		},
-		{
-			ViewName:    "main",
```

Comment:
pretty sure people weren't using these keybindings. Admittedly a flaw of our current controller-context separation of concerns is that it gets annoying to share methods between controllers